docs/yaml.md

```diff
@@ -88,14 +88,14 @@ build:
 
 The name given to built Docker images. If you want to push to a registry, this should also include the registry name.
 
-If you don't provide this, a name will be generated from the directory name.
-
 For example:
 
 ```yaml
 image: "registry.hooli.corp/jazz-solo-model"
 ```
 
+If you don't provide this, a name will be generated from the directory name.
```

Comment:
I moved the examples code blocks up above these caveat/gotcha blurbs so the examples won't be misunderstood as being related to those caveats. Does that make sense?

```
NAME
DESCRIPTION
EXAMPLE
CAVEATS
```

Reply:
Great idea.